pkg/didcomm/transport/http/inbound.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package http
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// InboundDIDCommHandler struct to pass details for request handlers
+type InboundDIDCommHandler struct {
```

Comment:
where is this struct type used externally? I see an internal usage within NewInboundHandler but this type is not returned.

Reply:
struct is now not exported